### PR TITLE
Newsletter 251 : translate in French

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin !

File: _posts/fr/newsletters/2023-05-17-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+## En attente de confirmation #1 : pourquoi avons-nous un mempool ?
+
+_Le premier segment d'une série hebdomadaire limitée sur le relais de transaction,
+l'inclusion dans le mempool et la sélection des transactions minières--- y compris
+pourquoi Bitcoin Core a une politique plus restrictive que celle permise par le
+consensus et comment les portefeuilles peuvent utiliser cette politique de la
+manière la plus efficace._
+
+{% include specials/policy/en/01-why-mempool.md %}
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-05-17-newsletter.md:160]: @Copinmalin Did you plan on translating this section as well?

Comment (bitschmidty) [reply _posts/fr/newsletters/2023-05-17-newsletter.md:160]: @Copinmalin when you are done, you can change this link accordingly

Comment (Copinmalin) [reply _posts/fr/newsletters/2023-05-17-newsletter.md:160]: OK, done !

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin !

**Comment by Copinmalin** [comment]
Thanks :-)

**Pull request merged**